netbox/dcim/models/device_components.py

```diff
@@ -115,26 +117,32 @@ def get_peer_port(termination):
 
             # Map a front port to its corresponding rear port
             if isinstance(termination, FrontPort):
-                position_stack.append(termination.rear_port_position)
                 # Retrieve the corresponding RearPort from database to ensure we have an up-to-date instance
                 peer_port = RearPort.objects.get(pk=termination.rear_port.pk)
+
+                # Don't use the stack for 1-on-1 ports, they don't have to come in pairs
```

Comment:
What does it mean to "come in pairs?"

Reply:
I have struggled to describe this succinctly (and apparently failed). The pairs I meant are:

`[FPx] [RP] --some-path-- [RP][FPx]` where multiplexing multiple FPs onto one RP needs to demux them again through an RP to an FP.  What would be better wording?